test/localstack.ts

```diff
@@ -157,6 +157,7 @@ export const localstackTestSuite = (
   testResource.skip = (name, resources, test) => {
     tests.push({ name, resources, test: test as any, skip: true, only: false });
   };
+  // eslint-disable-next-line no-only-tests/no-only-tests
   testResource.only = (name, resources, test) => {
```

Comment:
Why is the linter finding this? Does it alert on any call of .only?

Reply:
not sure, actually, the plugin claims to search this list by default: `["describe", "it", "context", "test", "tape", "fixture", "serial"]`

But it "fixed" that line :(

Reply:
Looks like the plugin is kind of dumb

https://github.com/levibuzolic/eslint-plugin-no-only-tests/blob/master/rules/no-only-tests.js#L65-L71